config/ePO-default/namespaces.xml

```diff
@@ -14,15 +14,15 @@
     <prefix name="skos" value="http://www.w3.org/2004/02/skos/core#"/>
 
     <prefix name="epo" value="http://data.europa.eu/a4g/ontology#"/>
+    <prefix name="epo-not" value="http://data.europa.eu/a4g/ontology#"/>
```

Comment:
good input for the moment. 

I fear we will need other namespace definitions too, at least for eOrdering, eFulfillment. 
you may as well add them now: "epo-cat", "epo-ful" (same base URI as above)